homeassistant/components/camera/__init__.py

```diff
@@ -888,9 +891,12 @@ async def websocket_update_prefs(
     changes.pop("id")
     changes.pop("type")
     entity_id = changes.pop("entity_id")
-    await prefs.async_update(entity_id, **changes)
-
-    connection.send_result(msg["id"], prefs.get(entity_id).as_dict())
+    try:
+        entity_prefs = await prefs.async_update(entity_id, **changes)
+        connection.send_result(msg["id"], entity_prefs)
```

Comment:
Please only wrap the line that we expect to raise in the try... except block. Sending the result shouldn't raise, right?

Reply:
Thanks Martin. I'll make a follow up PR to change this and the comment below.